cluster/kubernetes/update.go

```diff
@@ -34,7 +34,7 @@ func updatePodController(def []byte, newImageID flux.ImageID) ([]byte, error) {
 	}
 
 	var buf bytes.Buffer
-	err = tryUpdate(def, newImageID, &buf)
+	err = tryUpdate(def, stripNamespace(container), newImageID, &buf)
```

Comment:
(We figured this out IRL -- it was a bandaid over a problem with the test fixture)